Frontend/src/services/pointServices.js

```diff
@@ -59,7 +59,53 @@ function getPoint(pointId, callback){
     return null;
 };
 
+function updatePoint(point){
+    let content = {
+        'headers':{
+            'token': localStorage.getItem('token')
+        },
+        'body': point
+    }
+    let urlString = 'https://api.pointmap.net/api/point/'+point.Id
+    axios.put(urlString, content).then((response) => {
+        let data = response.data;
+        return data;
+        
+    }).catch((err) => {

```

Comment:
ping logout method authorization services, instead of doing it again, we need to refactor that still

Reply:
Also if you don't mind moving the urls used in the logout method in auth services in a url file that'd also be nice

Reply:
I'm placing deleteSession() in place of just locally deleting the token. I'm not sure if that is what you meant by this.